plugin-default-impl/src/main/java/com/alibaba/nacos/plugin/auth/impl/JwtTokenManager.java

```diff
@@ -97,7 +101,7 @@ public Authentication getAuthentication(String token) {
      * @param token token
      */
     public void validateToken(String token) {
-        Jwts.parserBuilder().setSigningKey(nacosAuthConfig.getSecretKeyBytes()).build().parseClaimsJws(token);
+        parser.parseClaimsJws(token);
```

Comment:
Whether JwtParser is thread safety?

Reply:
I write a utit test to check the thread safety issues.
`    @Test
    public void testConcurrentCreateTokenAndSecretKeyWithoutSpecialSymbol() {
        //this unit test is to approve the singleton jwtParse don't have thread safety issues
        IntStream.range(0, 100).parallel().forEach(i -> {
            try {
                createToken(null);
            } catch (NoSuchFieldException | IllegalAccessException | RuntimeException e) {
                e.printStackTrace();
                //concurrent parse jwt failed, make unit test failed.
                Assert.isTrue(false);
            }
        });
    }`
 'singleton jwtParse' parse  jwttoken success in multithreading environment , so this unit test approve   'singleton jwtParse'  don`t have a thread safety issues

Reply:
I debug the logic of JwtParser parse jwttoken,   JwtParser create a new SecretKeySpec object by clone the secretKeyBytes, so the  secretKey don`t have a thread safety issues.
![image](https://user-images.githubusercontent.com/8168864/157184831-d2c66d85-a9fc-41ea-937b-00bfe0982d4b.png)